.github/workflows/push.yml

```diff
@@ -10,16 +12,29 @@ env:
   TEST_AWS_SECRET_ACCESS_KEY: ${{ secrets.TEST_AWS_SECRET_ACCESS_KEY }}
   TEST_AWS_KMS_KEY_ID: c474f77b-65db-4778-975b-6f167e27d385
   TEST_AWS_ROLE_ID: arn:aws:iam::480319613404:role/ci-object-encryptor-role
+  TEST_GCP_KMS_KEY_ID: projects/kbc-ci-platform-services/locations/global/keyRings/ci-object-encryptor/cryptoKeys/ci-object-encryptor
 jobs:
   Build:
     runs-on: ubuntu-latest
     steps:
       - name: Check out the repo
         uses: actions/checkout@v3
+      - name: Google Login
+        uses: 'google-github-actions/auth@v1'
+        with:
+          credentials_json: ${{ secrets.TEST_GCP_SERVICE_ACCOUNT_KEY }}
+          export_environment_variables: true
+          create_credentials_file: true
+
       - name: Build image
         run: |
           docker login --username "$DOCKERHUB_USER" --password "$DOCKERHUB_TOKEN"
           docker-compose build
       - name: Run tests
         run: |
+          # GCP login generates `GOOGLE_APPLICATION_CREDENTIALS` environment variable, but for tests we need 
+          # `TEST_GOOGLE_APPLICATION_CREDENTIALS` which is assigned to the `GOOGLE_APPLICATION_CREDENTIALS` when needed.  
+          # https://github.com/google-github-actions/auth#other-inputs
+          export TEST_GOOGLE_APPLICATION_CREDENTIALS=$GOOGLE_APPLICATION_CREDENTIALS
```

Comment:
No a potrebujem to teda vubec prefixovat `TEST_`? Eventuelne nebylo by lepsi to namapovat v `docker-compose.yml`?

Reply:
je to stejný jako tohle https://github.com/keboola/object-encryptor/blob/83445f35fe912707b37ed5aebf7e5d2cf270ed01/.github/workflows/push.yml#L4

jinak ten prefix tady je protože se testuje exception handling při neplatnejch credentials - e.g. https://github.com/keboola/object-encryptor/blob/057a9dd324b78e2ee504971a165e1949787153a2/tests/GenericAKVWrapperTest.php#L366 což bych zrovna v téhle knihovně rád zachoval.

Reply:
Nejak nechapu jak to souvisi - v tom testu ty credentials explicitne nastavujes na "invalid". Ale to je jedno, jen me napadlo, jeslti by se toho nedalo zbavit, kdyz to potrebuje takovy comment :slightly_smiling_face: Nech to byt, jak to mas